Notepad/themes/system-minimal.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "System Minimal",
+    "style": "Light",
+    "author": {
+        "name": "Stefan Trauth",
+        "email": "mail@stefantrauth.de"
+    },
+    "version": "1.0",
+    "editor": {
+        "backgroundColor": "#FFFFFF",

```

Comment:
Also, could you remove the comma after this line?

Reply:
Done